src/status_im2/contexts/activity_center/events_test.cljs

```diff
@@ -1,163 +1,169 @@
 (ns status-im2.contexts.activity-center.events-test
```

Comment:
FYI: The diff in this file is somewhat big. The important point is that I was able to dramatically simplify them due to the simplicity of using `(is (= expected actual`) now that `:json-rpc/call` is just data in the file `activity_center/events.cljs`.